core/jvm/src/main/scala/fs2/compress.scala

```diff
@@ -75,197 +101,680 @@ object compress {
     */
   def inflate[F[_]](nowrap: Boolean = false, bufferSize: Int = 1024 * 32)(
       implicit ev: RaiseThrowable[F]
+  ): Pipe[F, Byte, Byte] = stream => {
+    val inflater = new Inflater(nowrap)
```

Comment:
Same as with `deflater` - wrap with `Stream.suspend`

Reply:
I'm not sure. Deflate and inflate return a function so a new Deflater is created for each use. I've added tests "deflate and inflate are reusable" and "gzip and gunzip are reusable." Let me know if I've misunderstood. Thanks

Reply:
The reply [here](https://github.com/functional-streams-for-scala/fs2/pull/1783#discussion_r376115069) was intended to be in response to this question. Sorry for the confusion!